### DISCUSSION TITLE
Audio Only Input?

### DISCUSSION THREAD

**Discussion by YenLegion** [blakeblackshear/frigate#7329] [General]
Curious about using microphones with a pi zero, or is that out of the scope of frigate?

**Comment by NickM-27** [comment]
audio only recordings are not supported and frigate still expects a video feed to decode.

if you created an mp4 video that was just a black square that could be used along with the audio, but that is a pretty hacky workaround

**Reply by ctml91** [reply]
With the latest frigate beta adding support for audio events, do you see support for audio only input coming at some point? I'd love to add audio feeds around the home for detection (dog barking, baby crying, etc) without the need to stream a bunch of video feeds over wifi 24/7.  Though I suppose the right types of devices for audio only streams would need to exist (e.g. options other than bluetooth).

**Reply by NickM-27** [reply]
If ffmpeg supports it then audio only is already supported.
